packages/broker/src/ConfigWizard.ts

```diff
@@ -164,12 +171,17 @@ export const getConfigFromAnswers = (answers: any): Config => {
     pluginNames.forEach((pluginName) => {
         const template = PLUGIN_TEMPLATES[pluginName]
         if (answers.selectPlugins && answers.selectPlugins.includes(pluginName)){
-            // the publishHttp plugin is special, it needs to be added to the config after the other plugins
-            if (pluginName === 'publishHttp') {
+            if (answers[`${pluginName}Port`] === template.port){
+                // user selected default value, enable without specifying the port
+                config.plugins![pluginName] = {}
+            } else if (pluginName === 'publishHttp') {
+                // the publishHttp plugin is special, it needs to be added to the config after the other plugins
+                config.plugins![pluginName] = {}
                 config.httpServer = {
                     port: answers[`${pluginName}Port`]
                 }
             } else {
+                // use provided a custom value, fill in
                 config.plugins![pluginName] = { ...template, port: answers[`${pluginName}Port`] }
```

Comment:
Is the `...template` part needed anymore?

Reply:
Ok 👍 Do we support that future feature for `publishHttp` too?

Reply:
not at the moment since it has it's own special path having to do with `httpServer`

Reply:
This may be confusing if someone assumes that the feature is working ok for all plugins. Maybe easiest is to drop that redundant feature

Reply:
Applied proposed solution